src/lib/containers/UserBadgeView.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+const uuidv4 = require("uuid/v4")
+
+const UserBadgeView = ({data, loadingBar}) => {
```

Comment:
Very nice, and since you have all of the profile data Ljubo can decide what should be shown (and style it).